src/web/plot.rkt

```diff
@@ -132,6 +132,38 @@
   (define info (regime-info altn))
   (and info (sp-bexpr (car info))))
 
+;;; Cost vs. Accuracy JSON (internal, single benchmark)
+(define (make-cost-accuracy-json result out)
+  (define repr (test-output-repr (test-result-test result)))
+  (define bits (representation-total-bits repr))
+  (define costs (test-success-end-costs result))
+  (define errs (map errors-score (test-success-end-errors result)))
+
+  (define cost0 (test-success-start-cost result))
+  (define err0 (errors-score (test-success-start-error result)))
+
+  (define xmax (argmax identity (cons cost0 costs)))
+  (define xmin (argmax identity (cons cost0 costs)))
+
+  (define json-obj `#hasheq(
+    (first . ,(list cost0 err0))
+    (best . ,(list (* 2 xmax) bits))
+    (points . ,
+      (for/list ([acost costs] [aerr errs]) (list acost aerr)))))
```

Comment:
Should there be a field for ymax and xmax, so it can be read in from the JS side?
Also confused about the `2 * xmax` calculation

Reply:
I think that would probably be better. Right now the only thing the "best x" and "best y" are being used for is the domain and range of the graph and having domain and range as separate fields is probably better.